SECURITY.md

```diff
@@ -1,6 +1,38 @@
 # Security Policy
 
-## Reporting a Vulnerability
+This page borrows parts of its contents from https://kubernetes.io/security/
 
-Please report vulnerabilities to <opensource@dynatrace.com>.
+## Report a Vulnerability
+
+We are extremely grateful for security researchers and users that report vulnerabilities to the Dynatrace Open Source Community. All reports are thoroughly investigated by a set of community members.
+
+To make a report, submit your vulnerability to <opensource@dynatrace.com>. This allows triage and handling of the vulnerability with standardized response times.
```

Comment:
rather than standardized response time, this is best effort

Reply:
would you say `appropriate timeframe` would fit your needs?

Reply:
yes